website/docs/r/tag_category.html.markdown

```diff
@@ -0,0 +1,107 @@
+---
+layout: "vsphere"
+page_title: "VMware vSphere: vsphere_tag_category"
+sidebar_current: "docs-vsphere-resource-inventory-tag-category"
+description: |-
+  Provides a vSphere tag category resource. This can be used to manage tag categories in vSphere.
+---
+
+# vsphere\_tag\_category
+
+The `vsphere_tag_category` resource can be used to create and manage tag
+categories, which determine how tags are grouped together and applied to
+specific objects.
+
+For more information about tags, click [here][ext-tags-general]. For more
+information about tag categories specifically, click
+[here][ext-tag-categories].
+
+[ext-tags-general]: https://docs.vmware.com/en/VMware-vSphere/6.5/com.vmware.vsphere.vcenterhost.doc/GUID-E8E854DD-AA97-4E0C-8419-CE84F93C4058.html
+[ext-tag-categories]: https://docs.vmware.com/en/VMware-vSphere/6.5/com.vmware.vsphere.vcenterhost.doc/GUID-BA3D1794-28F2-43F3-BCE9-3964CB207FB6.html
+
+~> **NOTE:** Tagging support is unsupported on direct ESXi connections and
+requires vCenter 6.0 or higher.
+
+[resource-vmfs-datastore]: /docs/providers/vsphere/r/vmfs_datastore.html
+
+## Example Usage
+
+This example creates a tag category named `terraform-test-category`, with
+single cardinality (meaning that only one tag in this category can be assigned
+to an object at any given time). Tags in this category can only be assigned to
+VMs and datastores.
+
+```hcl
+resource "vsphere_tag_category" "category" {
+  name        = "terraform-test-category"
+  description = "Managed by Terraform"
+  cardinality = "SINGLE"
+
+  associable_types = [
+    "VirtualMachine",
+    "Datastore",
+  ]
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (String, required) The name of the tag.
+* `description` - (String, optional) A description for the tag.
+* `cardinality` - (String, required, forces new resource) The number of tags
+  that can be assigned from this category to a single object at once. Can be
+  one of `SINGLE` (object can only be assigned one tag in this category), to
+  `MULTIPLE` (object can be assigned multiple tags in this category).
+* `associable_types` - (List of strings, required) A list object types that
+  this category is valid to be assigned to. For a full list, click
+  [here](#associable-object-types).
+
+~> **NOTE:** You can add associable types to a category, but you cannot remove
+them. Attempting to do so will result in an error.
```

Comment:
sounds like this should be `ForceNew`?

Reply:
_Adding_ types is a legitimate operation, removing them is not - so we can't be `ForceNew` here as that would block the case where someone wants to add a type to a category.